src/app/home/components/Header.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { hot } from 'react-hot-loader';
+import styled from 'styled-components';
+import logo from '../assets/logoBlanco.png';
+import leafLogo from '../assets/icons8-natural-food-50.png';
+import { Box, SubBox, ReciclandoLogo, LeafLogo, HeaderTitle} from '../styles/header';
```

Comment:
Seem to be that you are not using eslint and prettier, please enable on visual studio code